### ISSUE TITLE
Importing in the renderer on react  throws error

### ISSUE THREAD

**Issue by baraa-elwan** [hardchor/electron-redux#129]
When importing from electron-redux in the renderer it throws an error: "TypeError: fs.existsSync is not a function".
this problem was mentioned several times before see: #96 , #36 , #102 
After search on google the problem is :
'fs' is a node.js core module and webpack brings its own require which clobbers node.js' require, so when you require a node.js core module that webpack can't resolve to one of your files or dependencies, it throws.

error throws from electron module ./index.js file 
![Capture](https://user-images.githubusercontent.com/10418808/62459253-959be280-b787-11e9-9502-dc623124f37a.PNG)

this problem was appeared when require entire electron module 

**To solve the problem:**
instead requiring electron module,  require single module that used, e.g. in ./dist/middleware/forwardToMain.js
I changed this lines
```
//line 9
--var __electron= require("electron");
++var {ipcRenderer} =(typeof window != "undefined") ?  window.require("electron"): require("electron");

//line 33
--     __elecron.ipcRenderer.send('redux-action', action);
++   ipcRenderer.send('redux-action', action);
```

**Comment by pellejacobs** [comment]
This would seem like a clean solution. We're using webpack to bundle the renderer, and this seems to be the preferred way to work around this: https://github.com/electron/electron/issues/9920#issuecomment-313827210. 

@hardchor would you accept a PR with these changes?

**Comment by hardchor** [comment]
@baraa-elwan @pellejacobs I unfortunately have no way of testing this. Could you try this change out in your own projects and let me know if it helps fix your issue, please?

**Comment by hardchor** [comment]
@baraa-elwan I just published `1.5.1`. Please check if this fixes your issue and feel free to reopen if it doesn't.
